src/runtime/x86_cpu_features.cpp

```diff
@@ -79,6 +81,11 @@ WEAK CpuFeatures halide_get_cpu_features() {
             }
             if ((info2[1] & avx512_cannonlake) == avx512_cannonlake) {
                 features.set_available(halide_target_feature_avx512_cannonlake);
+                if ((info2[2] & avx512vnni) == avx512vnni) {
+                    // FIXME: Should also check AVX512-BF16 here, but that needs call to cpuid(eax=7, ecx=1)
```

Comment:
Suggestion: create a GitHub issue for "Implement SapphireLake features" to track the work to be done, and reference it here via `// TODO: (github-issue-link)`